packages/apollo-server-fastify/src/ApolloServer.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
 import { IncomingMessage, OutgoingMessage, ServerResponse, Server } from 'http';
 import { graphqlFastify } from './fastifyApollo';
-import { GraphQLOperation } from 'graphql-upload';
+import type { GraphQLOperation } from '@apollographql/graphql-upload-8-fork';
```

Comment:
Pretty sure what you've done here and the corresponding change in `package.json` is correct, though I'm having a hard time saying with certainty. Just noting since you also had a question about it and I don't want to say "yes" without saying "I'm not totally sure".

Reply:
Well two people at "pretty sure" equals a "totally sure", right?